src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-weight: 900;
+}
+
+.data-form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  background: url(images/Search.svg) no-repeat;
+  background-color: #fff;
+  outline: none;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
```

Comment:
Most of this properties are duplicated, only box-shadow is different. DRY

Reply:
Yeah, today we had have a reminder to watch closely what we add from Figma 😅